data/data-pipeline/data_pipeline/etl/sources/epa_rsei_aggregate/etl.py

```diff
@@ -0,0 +1,268 @@
+from pathlib import Path
+import math
+import pandas as pd
+
+from data_pipeline.etl.base import ExtractTransformLoad
+from data_pipeline.score import field_names
+from data_pipeline.utils import get_module_logger, unzip_file_from_url
+
+logger = get_module_logger(__name__)
+
+
+class EPARSEISCOREETL(ExtractTransformLoad):
```

Comment:
Same comment as on other PR: can we make this class name more readable? They're supposed to be CamelCase but the abbreviations throw it off.

Reply:
Like `RiskScreeningEnvironmentalIndicatorsETL` or something. Could also rename the `epa_rsei_aggregated_microdata` references to `epa_risk_screening_environmental_indicators`.

Reply:
How about `epa_rsei_aggregated_risk_environmental_indicator`? This ensures there is still specificity of the data. There is also disaggregated and other geographic units of measurements provided.

Reply:
Understood - I will make the change.